### ISSUE TITLE
Jacoco coverage report won't reflect latest change

### ISSUE THREAD

**Issue by jason1peng** [Kotlin/kotlinx-kover#489]
**Describe the bug**
Jacoco report was not update correctly after delete one of the tests.
Coverage report still showing object is covered.

**Errors**
-

**Expected behavior**
Coverage report should show object is not covered.

**Reproducer**
Basic project with two test file.

build.gradle.kts
```
plugin {
  id("org.jetbrains.kotlinx.kover") version "0.7.4"
}
...
kover {
    useJacoco("0.8.10")
}
```
Target class
```
object HelloWorld {
    fun hello() = "Hello"
    fun world() = "${hello()} World"
}
```
Test file 1
```
class HelloTest : DescribeSpec({
    describe("hello") {
        context("hello") {
            it("should return hello") {
                HelloWorld.hello() shouldBe "Hello"
            }
        }
    }
})
```
Test file 2
```
import io.kotest.core.spec.style.DescribeSpec
import io.kotest.matchers.shouldBe

class WorldTest : DescribeSpec({

    describe("world") {
        context("world") {
            it("should return hello world") {
                HelloWorld.world() shouldBe "Hello World"
            }
        }
    }

})
```

* trigger koverHtmlReport task, I could see HelloWorld class is covered 100%.
* delete test file 2, trigger koverHtmlReport task, HelloWorld class still covered 100%.

**Reports**
-

**Environment**
 - Kover Gradle Plugin version: 0.7.4
 - Gradle version: 7.6.1
 - Kotlin project type: Kotlin
 - Coverage Toolset (if customized in build script): JaCoCo("0.8.10")
 - Other context important for this bug: mac

**Comment by jason1peng** [comment]
Thanks for the quick update. I tried these two approaches and both won't work.
1. run `gradle clean` before generating report. 
2. use `--no-build-cach`
Just try to provide as much as I can to make sure problem could be fix in proper way.

**Comment by shanshin** [comment]
Fixed in `0.7.5`
